x11-libs/goffice/goffice-0.10.43.ebuild

```diff
@@ -2,13 +2,14 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI="6"
+
 GNOME2_LA_PUNT="yes"
 
 inherit gnome2
 
 DESCRIPTION="A library of document-centric objects and utilities"
 HOMEPAGE="https://git.gnome.org/browse/goffice/"
-SRC_URI="https://ftp.gnome.org/pub/GNOME/sources/goffice/0.10/goffice-0.10.43.tar.xz"
+SRC_URI="https://ftp.gnome.org/pub/GNOME/sources/goffice/0.10/${P}.tar.xz"
```

Comment:
`../${PN}/${PV%.*}/${P}...`

Reply:
Not sure why you added SRC_URI really, it is generated by gnome.org eclass which is inherited by gnome2.eclass